website/src/docs/hotchocolate/defining-a-schema/operations.md

```diff
@@ -370,15 +330,242 @@ public class DefaultTransactionScopeHandler : ITransactionScopeHandler
 }
 ```
 
-If you implement a custom transaction scope handler or if you choose to extend upon the default transaction scope handler you can add it like the following.
+If we implement a custom transaction scope handler or if we choose to extend upon the default transaction scope handler, we can add it like the following.
 
 ```csharp
 services
     .AddGraphQLServer()
-    ...
+    // ...
     .AddTransactionScopeHandler<CustomTransactionScopeHandler>();
 ```
 
 # Subscription
 
-Documentation following ...
+The subscription type in GraphQL is used to add real-time capabilities to our applications. Clients can subscribe to events and receive the event data in real-time, as soon as the server publishes it.
+
+Subscribing to an event is like writing a standard query. The only difference is the operation keyword and that we are only allowed to have one root field.
+
+```graphql
+subscription {
+  onBookAdded(author: "Jon Skeet") {
+    title
+  }
+}
+```
+
+HotChocolate implements Subscriptions via WebSockets and uses the pub/sub approach of [Apollo](https://www.apollographql.com/docs/apollo-server/data/subscriptions/#the-pubsub-class) for triggering events.
+
+## Defining a Subscription
+
+A subscription type can be represented like the following:
+
+<ExampleTabs>
+<ExampleTabs.Annotation>
+
+```csharp
+public class Subscription
+{
+    [Subscribe]
+    public Book OnBookAdded([EventMessage] Book book) => book;
+}
+
+public class Startup
+{
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services
+            .AddGraphQLServer()
+            // ...
+            .AddSubscriptionType<Subscription>();
+    }
+
+    // Omitted code for brevity
+}
+```
+
+</ExampleTabs.Annotation>
+<ExampleTabs.Code>
+
+```csharp
+public class SubscriptionType : ObjectType
+{
+    protected override void Configure(IObjectTypeDescriptor descriptor)
+    {
+        descriptor
+            .Field("onBookAdded")
+            .Type<PersonType>()
+            .Resolve(context => context.GetEventMessage<Person>())
+            .Subscribe(async context =>
```

Comment:
This code works, but is this how you would do it? I haven't used subscriptions like this before...